File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/AccountInfo.java
package kz.greetgo.sandbox.controller.model;

public class AccountInfo {
  public int id;
  public String fullName;
  public String charm;
  public int age;
  public float totalAccBalance;
  public float maxAccBalance;
  public float minAccBalance;

  public AccountInfo(int id, String fullName, String charm, int age, float totalAccBalance, float maxAccBalance, float minAccBalance) {
    this.id = id;
    this.fullName = fullName;
    this.charm = charm;
    this.age = age;
    this.totalAccBalance = totalAccBalance;
    this.maxAccBalance = maxAccBalance;
    this.minAccBalance = minAccBalance;
  }

  public AccountInfo(String fullName) {
    this.fullName = fullName;
  }

  public AccountInfo() {

  }
}

File: sandbox.client/front/ts/model/AccountInfo.ts
export class AccountInfo {
  public id: number/*int*/;
  public fullName: string;
  public charm: string;
  public age: number/*int*/;
  public totalAccBalance: number /*float*/;
  public maxAccBalance: number/*float*/;
  public minAccBalance: number/*float*/;

  public assign(o: any): AccountInfo {
    this.fullName = o.fullName;
    this.charm = o.charm;
    this.age = o.age;
    this.totalAccBalance = o.totalAccBalance;
    this.maxAccBalance = o.maxAccBalance;
    this.minAccBalance = o.minAccBalance;
    return this;
  }

  public static copy(a: any): AccountInfo {
    let result = new AccountInfo();
    result.assign(a);
    return result;
  }
}
